src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,86 @@
+.product-card {
+  box-sizing: border-box;
+  height: 406px;
+  width: 200px;
+  padding: 206px 16px 16px;
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position: 19px 32px;
+  border: 1px solid#F3F3F3;
+  border-radius: 5px;
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__code-product {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0 0 16px;
+    color: $second-color;
+  }
+
+  &__reviews {
+    margin-bottom: 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__reviews-paragraf {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-first {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $second-color;
```

Comment:
Variables should be used for font sizes, line heights, and colors to ensure consistency and easy maintenance. Define these values in `variables.scss` and use the variables here.